wp-content/themes/core/pcss/components/_dialog.pcss

```diff
@@ -0,0 +1,87 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog[aria-hidden='true'] {
+	display: none;
+}
+
+.c-dialog__overlay {
+	background-color: var(--color-goethite);
+	bottom: 0;
+	color: var(--color-white);
+	left: 0;
+	overflow: scroll;
+	position: fixed;
+	right: 0;
+	top: 0;
+	width: 100%;
+	z-index: 2;
+
+	.admin-bar & {
+		top: var(--spacer-wpadmin-mobile);
+
+		@media (--viewport-wpadmin) {
+			top: var(--spacer-wpadmin-desktop);
+		}
+	}
+}
+
+.c-dialog__header {
+	background-color: var(--color-goethite);
+	border-bottom: 1px solid var(--color-white);
+	margin-bottom: 47px;
+	padding: 20px;
+	position: fixed;
+	width: 100%;
+	z-index: 10;
+}
+
+.c-dialog__title {
+	margin-bottom: 0;
+	padding: 0 20px;
+	text-align: center;
+}
+
+.c-dialog__content-wrapper {
+	padding-top: 120px;
+
+	@media (--viewport-medium) {
+		padding-top: 140px;
+		width: 100%;
+	}
+}
+
+.c-dialog__content-inner {
+	margin: 0 auto;
+	padding: 0 20px;
+	max-width: 1000px;
```

Comment:
Let's either add this to the other grid vars as an available max-width or pick one that exists that is close enough